weblate/trans/views/create.py

```diff
@@ -520,10 +520,20 @@ def form_valid(self, form):
             return self.redirect_create(
                 repo=component.get_repo_link_url(),
                 project=component.project.pk,
+                category=component.category.pk,
```

Comment:
Category can be `None` and then this fails with: `AttributeError: 'NoneType' object has no attribute 'pk'`